resources/manifests/pod-checkpointer-psp.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: pod-checkpointer-restricted
```

Comment:
I think this is quite tricky. It is worth a comment to explain (or link to) the PSP policy order and the reasoning for the chosen name

Reply:
fixed